migrator/README.md

```diff
@@ -0,0 +1,261 @@
+# Migrator Tool
+
+The library takes in E.164 phone numbers with their corresponding BCP-47 country code. If inputted numbers match any of the supported internal renumbering ‘recipes’, which are created from official country documentation, said numbers will undergo a process of migration to convert them into the most up to date, dialable format. A given “recipe” is an internal record which holds data needed to migrate a stale number into one that is valid and dialable. 
+
+## Capabilities
+
+```Single Number Migration``` - input a single E.164 phone number with its corresponding BCP-47 country code. If there is an available migration recipe, the number will be converted to the new format based on the given recipe. The resulting string of the migration will be outputted to the console.
+
+```File Migration``` - input a text file location with one E.164 number per line along with the BCP-47 country code that corresponds to the numbers in the text file. All numbers in the text file that match available recipes will be migrated and a copy of the text file will be created with the updated numbers. The file path to the newly created text file will be outputted to the console.
+
+```Migration Report``` - find all phone numbers in a given input that were converted with given migration recipes. The report will also show the phone numbers that were already seen as valid before migrations as well as identify any numbers that were migrated into invalid formats based on known valid phone number ranges for the specified country code.
+
+```Custom Recipe Migrations``` - specify custom recipe files to use for migrations instead of the internal maintained file. Custom recipe migrations will not check if converted numbers are valid based on country formats. 
+
+```Web Application``` - As an alternative to using the tool from the command line, [click here](https://phone-number-migrator.uc.r.appspot.com/) to access the web application version. 
+
+## Required Parameters
```

Comment:
I also think this should be after installation and setup. Maybe there could be a big Usage section and then have the required parameters under it, as well as all the examples you have.